scripts/node/dictionary/formats.js

```diff
@@ -34,21 +34,25 @@ const jsComment = () =>
  */
 const formatJSCustomProperties = (dictionary) =>
   [
-    `${jsComment()}`,
-    'module.exports = {',
+    jsComment(),
+    'const cssVariables = {',
     ...dictionary.allProperties.map((prop) => `  "--${prop.name}": "${prop.value}",`),
     '};',
+    'export default cssVariables;',
   ].join('\n');
 
 /**
  * @param {Array} dictionary style-dicitonary dictionary
  * @return {String} js file with commonJS exports
  */
 const formatCommonJs = (dictionary) =>
```

Comment:
[should] I would remove all these references to CommonJs in here if this is no longer CommonJs